float64 minX
float64 maxX

float64 minY
float64 maxY
